test/recipes/80-test_ssl_old.t

```diff
@@ -549,14 +549,17 @@ sub testssl {
 	  }
 
 	SKIP: {
-	    skip "skipping PSK tests", 2
+	    skip "skipping PSK tests", 3
 	        if ($no_psk);
 
 	    ok(run(test([@ssltest, "-tls1", "-cipher", "PSK", "-psk", "abc123"])),
 	       'test tls1 with PSK');
 
 	    ok(run(test([@ssltest, "-bio_pair", "-tls1", "-cipher", "PSK", "-psk", "abc123"])),
 	       'test tls1 with PSK via BIO pair');
+
+            ok(run(test(['ssl_old_test', '-psk', '0102030405', '-cipher', '@SECLEVEL=2:DHE-PSK-AES128-CCM'])),
```

Comment:
Indentation seems off here

Reply:
Spaces vs tabs.  I've used spaces, the file has tabs.  The indentation looks fine when editing.

Reply:
I've pushed a de-tabbing commit.

Reply:
I reverted the de-tabbing in favour of #15824